blazor/chart/zooming.md

```diff
@@ -154,6 +154,49 @@ By default, zoom in, zoom out, pan, and reset buttons are available in the toolb
 
 ![Zooming Option in Blazor Column Chart Toolbar](images/zoom/blazor-column-chart-zoom-in-toolbar.png)
 
+### Toolbar Display Mode
+
+By default, zooming tool bar will appear only when the chart is zoomed. Now we can display zooming tool bar for un zoomed charts by setting the ToolbarDisplayMode as ToolbarMode.Always.
```

Comment:
By default, the chart zooming toolbar only appears when the chart is zoomed. Now, we can display the zooming toolbar initially while the chart is loading by setting the `ToolbarDisplayMode` to `ToolbarMode.Always`.

Reply:
changed